packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -20,6 +20,8 @@ import {
   LINE_BREAKPOINT_INFO_REQUEST
 } from './constants';
 import { nls } from './messages';
+import pathExists = require('path-exists');
```

Comment:
Can this be `import * as pathExists from 'path-exists';`?

Reply:
Done.